modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/editor/configuration/JournalArticleContentEditorConfigContributor.java

```diff
@@ -57,6 +61,19 @@ public void populateConfigJSONObject(
 		}
 
 		jsonObject.put("removePlugins", removePlugins);
+
+		PortletDisplay portletDisplay = themeDisplay.getPortletDisplay();
+
+		if (Validator.isNotNull(portletDisplay.getId())) {
+			PortletURL portletURL = PortletURLBuilder.create(
+				requestBackedPortletURLFactory.createActionURL(
+					JournalPortletKeys.JOURNAL)
+			).setActionName(
+				"/journal/upload_image"
+			).buildString();
```

Comment:
This does not look right. `.buildString()` returns a string, not a `PortletURL`. You can either have a string variable, or use `.buildPortletURL()`.